legal-api/src/legal_api/reports/report.py

```diff
@@ -211,9 +211,6 @@ def _get_template_filename(self):
             # Get template specific to legal type
             file_name = None
             specific_template = ReportMeta.reports[self._report_key].get(self._business.legal_type, None)
-            if self._business.legal_type == 'CP' and self._filing.filing_type == 'correction':
-                if is_special_resolution_correction_by_meta_data(self._filing):
-                    file_name = 'specialResolutionCorrectionApplication'
```

Comment:
You're still going to need this

Reply:
As discussed, I think we want to move the `specialResolutionCorrectionApplication` pdf contents into the `correction` pdf